### ISSUE TITLE
It seems that the 2.0.3 version of SQLitePCLRaw.bundle_winsqlite3 will cause UWP to fail to activate in Release mode

### ISSUE THREAD

**Issue by zhuxb711** [ericsink/SQLitePCL.raw#346]
If I use version 2.0.2 of SQLitePCLRaw.bundle_winsqlite3, it works fine. 

If you use version 2.0.3, it will cause UWP to fail to activate normally at startup

**Comment by ericsink** [comment]
Hmmm.

My first guess is that winsqlite3 doesn't support `sqlite3_stmt_isexplain`, which was introduced in SQLite 3.28.0

**Comment by Shad0wlife** [comment]
Can confirm, version 2.0.3 causes .NET Native builds to throw a 0xc0000139 entry point not found error. 2.0.2 works fine.

**Comment by zhuxb711** [comment]
@Shad0wlife Do you have a sample program that can reproduce this problem ?

**Comment by Shad0wlife** [comment]
Setting up a repo for this would be way overkill already.
This is what you need to do to reproduce the issue:

1. Create a new UWP Project with min version Fall Creators Update 16299 (to enable .NET Standard compatibility) 
2. add [SQLitePCLRaw.bundle_winsqlite3](https://www.nuget.org/packages/SQLitePCLRaw.bundle_winsqlite3/) via NuGet
3. try to run the project in a configuration with the .NET Native Toolchain enabled (eg. a Release-Build with default settings) will cause the program start to fail. The 0xc0000139 error can then be seen in the program output window
4. for reference, disable the .Net Native Toolchain in the build settings for the release build (or just use a debug build), the program will start fine

Hope this helps!

**Comment by zhuxb711** [comment]
@Shad0wlife 
See this:
https://developercommunity.visualstudio.com/content/problem/1019173/unhanle-exception-0xe0434352-in-clrdll.html

**Comment by Shad0wlife** [comment]
Just a follow up because I recently looked into the cause for thi issue again:

I'm currently on Win10 1809 for compatibility reasons, and the winsqlite3.dll is of version 3.23.2 (according to file details).
[The "sqlite3_stmt_isexplain" function was only added in v3.28.0](https://www.sqlite.org/changes.html), so that's the reason it is not available in winsqlite. Bundled e_sqlite obviously doesn't have this problem, since it does not depend on Win10's sqlite.

So basically what I'm saying is: this issue was caused by Win10 having an outdated version of sqlite included (at least in older versions), so newer builds of Windows may have this issue fixed by default.
